### ISSUE TITLE
Training for a new language

### ISSUE THREAD

**Issue by tchayintr** [myshell-ai/MeloTTS#68]
Hi, thanks for your great work!

I would like to clarify if https://github.com/myshell-ai/MeloTTS/blob/main/docs/training.md supports training for a new language.

Plus, do you have any tips when training on a new language from your experience with this project?

Thanks again!

**Comment by jeremy110** [comment]
In my case, i use ipa to training a new language.

You need to 
1. change the bert model for your language
2. change g2p code 
3. add tones and add some your language variables in symbols.py

**Comment by jadechip** [comment]
@jeremy110 could you share how much training data you used and was it all of the same speaker?

**Comment by jeremy110** [comment]
@jadechip hello～
I using two datasets.
1. 1 speaker 10h
2. 14 speaker, each speakers 30 minutes 
training 300 epochs

**Comment by jadechip** [comment]
Thank you @jeremy110 for the insightful response, I really appreciate it!

**Comment by jeremy110** [comment]
@jadechip 
Don't metion it,You can also refer this https://github.com/myshell-ai/MeloTTS/issues/83

**Comment by jadechip** [comment]
That is super helpful, I'll report back with my code/results and training loss in case it's useful for others. 
Thanks again!

**Comment by Yongbi9** [comment]
@tchayintr please is it possible to train MeloTTS on a new language ?

**Comment by jeremy110** [comment]
@Yongbi9 
New discussion in https://github.com/myshell-ai/MeloTTS/issues/120

**Comment by tchayintr** [comment]
> @tchayintr please is it possible to train MeloTTS on a new language ?

@Yongbi9 Yes, it’s possible. I recommend to find more clues here: 
> @Yongbi9 New discussion in #120
